tests/journal/balance-assertions.test

```diff
@@ -308,3 +308,51 @@ hledger -f - stats
 >>> /Transactions/
 >>>2
 >>>=0
+
+# 17. Exact assertions parse correctly
+hledger -f - stats
+<<<
+2016/1/1
+    a      $1
+    b
+
+2016/1/2
+    a         == $1
+>>> /Transactions/
+>>>2
+>>>=0
+
+# 18. Exact assertions consider entire account
+hledger -f - stats
+<<<
+2016/1/1
+    a      $1
+    b
+
+2016/1/2
+    a       1 zorkmids
+    b
+
+2016/1/3
+    a       0 == $1
+>>>2 /balance assertion error.*line 10, column 15/
+>>>=1
+
+# 19. Mix different commodities and exact assignments
+hledger -f - stats
+<<<
+2016/1/1
+    a      $1
+    a      -1 zorkmids
+    b
+
+2016/1/2
+    a     == $1
```

Comment:
How does this work.. isn't a $1 - 1 zorkmids at this point ?

Reply:
That's an assignment, not an assertion; the `-1 zorkmids` is being removed from the account.  I've updated the `b` posting to make that more clear.